src/nupic/algorithms/SvmT.hpp

```diff
@@ -730,7 +730,7 @@ inline float svm<traits>::rbf_function(float* x, float* x_end, float* y) const
 {
   float sum = 0;
 
-#if defined(NTA_ASM) && defined(NTA_PLATFORM_win32) && defined(NTA_COMPILER_MSVC)
+#if defined(NTA_ASM) && defined(NTA_ARCH_32) && defined(NTA_OS_WINDOWS) && defined(NTA_COMPILER_MSVC)
```

Comment:
I don't understand why the `&& defined(NTA_ARCH_32)` is here. It doesn't seem consistent with the other conditions where `defined(NTA_PLATFORM_win32)` was replaced with `defined(NTA_OS_WINDOWS)`.